python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -149,7 +149,7 @@ def __new__(
             run_config=check.opt_mapping_param(
                 convert_config_input(run_config), "run_config", key_type=str
             ),
-            tags=check.opt_mapping_param(tags, "tags", key_type=str, value_type=str),
+            tags=check.opt_mapping_param(tags, "tags", key_type=str),
```

Comment:
This needs to be wrapped in a call to `validate_tags` (from `dagster._core.definitions.utils`:

```
tags=validate_tags(check.opt_mapping_param(tags, "tags", key_type=str), allow_reserved=False)
```

Also the `tags` type annotation in the `__new__` signature should be changed to `Mapping[str, Any]`